src/Microsoft.ApplicationInsights.AspNetCore/Extensibility/Implementation/Tracing/AspNetEventSource.cs

```diff
@@ -3,6 +3,9 @@
 //     Copyright (c) Microsoft Corporation. All rights reserved.
 // </copyright>
 //-----------------------------------------------------------------------
+
+using System.Reflection;
```

Comment:
Can this using go inside the namespace for consistency?  I don't think it is needed for anything outside the namespace.